html/pfappserver/lib/pfappserver/Model/Node.pm

```diff
@@ -302,16 +303,16 @@ See pf::import::nodes
 =cut
 
 sub importCSV {
```

Comment:
The importCSV logic should be moved to pf::import::nodes to improve code sharing.

Reply:
Done